test/unit/math/prim/scal/meta/return_type_test.cpp

```diff
@@ -15,3 +20,64 @@ TEST(MetaTraits, ReturnTypeFloat) {
 TEST(MetaTraits, ReturnTypeInt) {
   test::expect_same_type<double, return_type<int>::type>();
 }
+
+TEST(MetaTraits, ReturnTypeVar) {
```

Comment:
Ahh... so you've hit the confusing part of our source layout. The `var` tests belong in the `rev` folder. The `fvar<double>` tests belong in the `fwd` folder. The  `fvar<fvar<var>>` tests belong in the `mix` folder.

I can help. Mind giving me permissions on your branch and I'll push directly there?

Reply:
Easy. I see what I did wrong now. I'll fix all of that.